src/library_browser.js

```diff
@@ -1164,6 +1164,12 @@ var LibraryBrowser = {
       GL.newRenderingFrameStarted();
 #endif
 
+      // If the current GL context is an OffscreenCanvas, but it was initialized with implicit swap mode, perform the swap
+      // in behalf of the user.
+      if (typeof GL !== 'undefined' && GL.currentContext && !GL.currentContext.attributes.explicitSwapControl && GL.currentContext.GLctx.commit) {
```

Comment:
This might happen on the main thread too, if `commit` exists? When does it exist?